services/core-web/src/constants/routes.ts

```diff
@@ -260,14 +260,6 @@ export const EDIT_PROJECT = {
   helpKey: "Edit-Project",
 };
 
-export const MAJOR_MINE_APPLICATION = {
-  route: "/pre-applications/:projectGuid/major-mine-application/:mmaGuid/:tab",
-  dynamicRoute: (projectGuid, mmaGuid, tab = "final-app") =>
-    `/pre-applications/${projectGuid}/major-mine-application/${mmaGuid}/${tab}`,
-  component: MajorMineApplicationTab,
-  helpKey: "Major-Mine-Application",
-};
-
```

Comment:
this came up when I did a quick search for "final-app" and it turns out it's not referenced anywhere. So I deleted it.